src/plugins/index_pattern_management/public/components/create_index_pattern_wizard/lib/get_indices.test.ts

```diff
@@ -43,6 +43,15 @@ const successfulSearchResponse = {
   },
 };
 
+const partialSearchResponse = {
+  rawResponse: {
```

Comment:
This is not the full and correct format for a partial response.
Your complete response format is missing important arguments (`isPartial` and `isRunning`) 
I would want to test this with an accurate response.